components/server/src/prebuilds/prebuild-manager.ts

```diff
@@ -115,6 +117,38 @@ export class PrebuildManager {
         }, opts);
     }
 
+    /**
+     * getRecentWebhookEvent checks if the webhook integration is active for the given user and project by querying the webhook event database and seeing if for the latest commit on the repository there exists a webhook event.
+     */
+    public async getRecentWebhookEvent(
```

Comment:
What about moving this into a different unit? Or maybe into the ConfigurationService directly?
My thinking is:
 - it's not really realted to Prebuilds on the code level
 - importing PrebuildManager already is problematic because it very easily creates dependency cycles. The more we can extract from it the better

Reply:
Moved in [`7bd89de` (#20228)](https://github.com/gitpod-io/gitpod/pull/20228/commits/7bd89de711bd3c0950a1f83b3ff7143cf3502d57)